stdlib/public/runtime/Errors.cpp

```diff
@@ -48,9 +48,11 @@
 #include <cxxabi.h>
 #endif
 
-#if SWIFT_SUPPORTS_BACKTRACE_REPORTING
-// execinfo.h is not available on Android. Checks in this file ensure that
-// fatalError behaves as expected, but without stack traces.
+#if SWIFT_SUPPORTS_BACKTRACE_REPORTING && !defined(_WIN32)
```

Comment:
Please use `__has_include(<excinfo.h>)` instead

Reply:
Fair enough. I removed the comment too sense it doesn't really make sense anymore.